__tests__/news.test.js

```diff
@@ -48,3 +49,41 @@ describe("/api/topics", () => {
     });
   });
 });
+
+//Test 3 - Get articles by ID
+describe("/api/articles/:article_id", () => {
+  describe("GET Article object by specific article ID", () => {
+    test("200: responds with an Object of a specific article", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(() => {
+          expect.objectContaining({
+            author: "butter_bridge",
+            title: "Living in the shadow of a great man",
+            article_id: 1,
+            body: "I find this existence challenging",
+            topic: "mitch",
+            created_at: 1594329060000,
+            votes: 100,
+          });
```

Comment:
*what* are you expecting to be this object? 🤔

You're getting a false-positive here

You **must** see a failing test before you solve it!